app/upgrades/vNEXT.go

```diff
@@ -53,6 +62,17 @@ var Upgrade_NEXT = Upgrade{
 		// Ref: https://github.com/pokt-network/poktroll/compare/vPREV..vNEXT
 
 		return func(ctx context.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+			logger := cosmostypes.UnwrapSDKContext(ctx).Logger()
+
+			// Adds new authz that were previously incorrect. See #1425
+			grantAuthorizationMessages := []string{
```

Comment:
@bryanchriswhite For future reference.

Note that this is critical for authorizations to work.